CMakeLists.txt

```diff
@@ -3,12 +3,12 @@ project (nfstrace)
 
 # check compiler and packages ==================================================
 if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU")
-    if (CMAKE_CXX_COMPILER_VERSION VERSION_LESS 4.8)
-        message (FATAL_ERROR "GCC version must be at least 4.8")
+    if (CMAKE_CXX_COMPILER_VERSION VERSION_LESS 6.0)
+        message (FATAL_ERROR "GCC version must be at least 6.0")
     endif ()
 elseif ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
-    if (CMAKE_CXX_COMPILER_VERSION VERSION_LESS 3.3)
-        message (FATAL_ERROR "Clang version must be at least 3.3")
+    if (CMAKE_CXX_COMPILER_VERSION VERSION_LESS 3.8)
+        message (FATAL_ERROR "Clang version must be at least 3.8")
```

Comment:
decrease these requirements to GCC 5.1 anf Clang 3.5
http://en.cppreference.com/w/cpp/compiler_support